starport/services/serve/serve.go

```diff
@@ -488,24 +498,35 @@ func (s *Serve) watchAppFrontend(ctx context.Context) error {
 			),
 			step.New(
 				step.Exec("npm", "run", "serve"),
+				step.Env(
+					fmt.Sprintf("HOST=%s", host),
+					fmt.Sprintf("PORT=%s", port),
+					fmt.Sprintf("VUE_APP_API_COSMOS=%s", xurl.HTTP(conf.Servers.APIAddr)),
```

Comment:
For some reasons these env vars does not reflect to @tendermint/vue. @fadeev any ideas?

Reply:
Are you talking about `HOST` and `POST`? Vue CLI only registers env variables starting with `VUE_APP_`.

Reply:
No actually, I'm talking about these:
```
VUE_APP_API_COSMOS
VUE_APP_API_TENDERMINT
VUE_APP_WS_TENDERMINT
```